app/controllers/questions/backtaxes_controller.rb

```diff
@@ -42,6 +42,7 @@ def form_params
         source: current_intake.source || source,
         referrer: current_intake.referrer || referrer,
         locale: I18n.locale,
+        visitor_id: visitor_id
```

Comment:
Would love to chat about this - I'm under the impression that we need this here because we get this information from the cookies. That said, I could be misled.